### ISSUE TITLE
Layout the TADAShiny Filter options similar to the layout used in Water Quality Portal

### ISSUE THREAD

**Issue by JamesBisese** [USEPA/TADAShiny#148]
**Is your feature request related to a problem? Please describe.**

There are 14 UI elements that make up the Filter options in TADA web app.  These all have 1 to 1 correspondance to UI elements in the Water Quality Portal.  It would be easier on the user community to have our UI elements displayed in similar order to WQP.

**Describe the solution you'd like**

The layout of the UI elements is easy to modify in a single file (mod_query_data.R).  The rows and columns used to place the UI elements could be reworked to make them match the WQP.

**Describe alternatives you've considered**

This may be unnecessary, but it takes away a bit of fiddling on our part.  Let WQP deal with the users about where items should be, and we just follow.

**Additional context**

Here is the WQP filter elements (under Advanced)

![image](https://github.com/USEPA/TADAShiny/assets/16490554/8d8f7bc7-cd2c-4e5f-a57e-e4f06ceb9f97)

and

![image](https://github.com/USEPA/TADAShiny/assets/16490554/41db26a2-93fd-492f-9bd7-888bb18d12d4)

and here is TADAShiny with the screen small but not yet at any breakpoint

![image](https://github.com/USEPA/TADAShiny/assets/16490554/747922d2-d3da-4a3c-be00-da685a9abab8)

**Reminders for TADA contributors addressing this issue**

New features should include all of the following work:

-   [ ] Create the function/code.

-   [ ] Document all code using comments to describe what is does.

-   [ ] Create tests.

**Comment by cristinamullin** [comment]
I agree but let's wait until the new WQX 3.0 profiles and UI is released and match that.

**Comment by JamesBisese** [comment]
This got done by Cristina 2026-05-29.  
https://github.com/USEPA/TADAShiny/commit/72784022b2469d084f922cbfe1b60bdc103c54e1
